src/clientStream.cc

```diff
@@ -110,7 +110,7 @@ clientStreamNode::~clientStreamNode()
  */
 void
 clientStreamInit(dlink_list * list, CSR * func, CSD * rdetach, CSS * readstatus,
-                 ClientStreamData readdata, CSCB * callback, CSD * cdetach, ClientStreamData callbackdata,
+                 ClientStreamData readdata, CSCB * callback, CSD * cdetach, const ClientStreamData &callbackdata,
```

Comment:
If possible, please avoid readdata copies as well (Coverity defect 1529543?).

Reply:
done

Reply:
Thank you. To reflect your changes, I have added CID 1529543 to the PR description.